code/datums/security_state.dm

```diff
@@ -210,7 +210,7 @@
 	overlay_alarm = "alarm_green"
 	overlay_status_display = "status_display_green"
 
-	down_description = "All threats to the station have passed. Security may not have weapons visible, privacy laws are once again fully enforced."
+	down_description = "All threats to the ship have passed. Security may not have weapons visible, privacy laws are once again fully enforced."
```

Comment:
This is a change that might not sit well with the plans to host Glloydstation on odd days. Since the alert will refer the station as a ship instead. How about we make another addition to the map datum, so we could use `[USING_MAP.vessel_type_name]` in place of station or ship instead?

Unfortunately. These variables expect those strings to be constants. So we would need to put them in functions or use regex to initialize those strings proper.

Reply:
Good shout and idea! Added in 7998729